aws/resource_aws_config_aggregator.go

```diff
@@ -0,0 +1,193 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/configservice"
+
+	"github.com/hashicorp/terraform/helper/customdiff"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsConfigAggregator() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsConfigAggregatorPut,
+		Read:   resourceAwsConfigAggregatorRead,
+		Update: resourceAwsConfigAggregatorPut,
+		Delete: resourceAwsConfigAggregatorDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		CustomizeDiff: customdiff.Sequence(
+			customdiff.ForceNewIfChange("account_aggregation_source", func(old, new, meta interface{}) bool {
```

Comment:
I was trying to minimise delete/recreate operations - it's OK to call `PutConfigurationAggregatorInput` to make changes to an existing aggregator as long as you aren't switching between organization or account aggregation sources.

Reply:
Ah: its because of this behavior (seemingly buggy as `ForceNew:` doesn't necessarily apply to all nested attributes anymore):

```
All fields are ForceNew or Computed w/out Optional, Update is superfluous
```

I'll add this in a comment in there.